src/tests/unit/test-data/foundry-list/foundry-list.expected

```diff
@@ -9,6 +9,7 @@ APRProof: test%AssertTest.checkFail_assert_false():0
     terminal: 3
     refuted: 0
     bounded: 0
+    execution time: 0h 0m 0s
```

Comment:
The execution time is incorrect.

Reply:
This is a unit test, so it's not actually executing anything, so this is OK. This test is just building a dummy proof object and testing that it lists correctly.